programs/cleos/main.cpp

```diff
@@ -538,6 +539,34 @@ void print_action_tree( const fc::variant& action ) {
    }
 }
 
+int get_return_code( const fc::variant& result ) {
+   // if a trx with a processed, then check to see if it failed execution for return value
+   int r = 0;
+   if (result.is_object() && result.get_object().contains("processed")) {
+      const auto& processed = result["processed"];
+      if( processed.is_object() && processed.get_object().contains( "except" ) ) {
+         const auto& except = processed["except"];
+         if( except.is_object() ) {
+            try {
+               auto soft_except = except.as<std::optional<fc::exception>>();
+               if( soft_except ) {
```

Comment:
At this point we've established that `except` is an object. Do we really need to treat `except` as an `optional` type still?

Reply:
No. Removed `optional`.